tests/infra/ccf.py

```diff
@@ -66,6 +66,49 @@ def __init__(self, hosts, dbg_nodes=None, perf_nodes=None, existing_network=None
         for host in hosts:
             self.create_node(host)
 
+    def _get_next_local_node_id(self):
+        if len(self.nodes):
+            return self.nodes[-1].node_id + 1
+        return self.node_offset
+
+    def create_node(self, host):
+        node_id = self._get_next_local_node_id()
+        debug = (
+            (str(node_id) in self.dbg_nodes) if self.dbg_nodes is not None else False
```

Comment:
self.dbg_nodes or []

It doesn't matter if dbg_nodes was None or an empty collection of any type.